docs/getting-started.md

```diff
@@ -69,6 +67,14 @@ Each combination of target account and region must be bootstrapped prior to depl
 cdk bootstrap
 ```
 
+Run the following command to confirm there are no issues with your code
+
+```sh
+npm run build 
+```
+
```

Comment:
This should not be necessary.

Reply:
resolved.

Update to remove above section about `npm run build` from both `README.md` and `getting-started.md` after discussion